### PR TITLE
Update hero copy

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
In short, I'm more fan of the messaging in our seed announcement. As such I've updated the hero copy to take the same line.

### CODE REVIEW

**Comment by joepavitt** [comment]
just flagging that you have two PRs open for the same thing here, I reviewed the other one then thought I was going crazy when I saw this too, both change the hero copy: https://github.com/flowforge/website/pull/244

**Comment by joepavitt** [comment]
My feedback on the hero copy can be found in #244

**Comment by ZJvandeWeg** [comment]
@joepavitt Thanks for flagging, staged a few too many changes. Should've used `git add -p`.

File: src/index.njk [hunk 1]
```diff
@@ -33,7 +33,7 @@ layout: default
 <div class="w-full ff-bg-light px-6 pt-12 pb-48 sm:pt-32 sm:px-0">
     <div class="container m-auto text-center max-w-4xl">
         <div class="m-auto mt-36 mb-12 sm:mt-64 sm:mb-16 w-full text-center">
-            <h2>Run Node-RED in Production</h2>
+            <h2>What FlowForge brings to Node-RED</h2>
```
Comment (ZJvandeWeg) [inline src/index.njk:36]: @joepavitt recommended the verb `add`. I'm not a fan of `add`, as I'd rather amplify and extend Node-RED. Add seems too much like Node-RED is forked IMO

**Review by knolleary — APPROVED** [review]

**Pull request merged**